main.c

```diff
@@ -3,26 +3,31 @@
 #include <time.h>
 
 int main() {
-    int tablica[50];
-    int rozmiarTablicy =
-            sizeof(tablica) / sizeof(tablica[0]);
-
-    double sredniaArytmetyczna = 0;
-    int i;
-    int number;
-    int j;
+    int array[50], i, j, arraySize = sizeof(array) / sizeof(array[0]);
+    double arithmeticAvg;
+    double arraySum = 0;
+
     srand(time(NULL));
-    printf("Wartosc tablicy:");
-    for (j = 0; j < rozmiarTablicy; j++) {
-        number = rand() % 50 + 0;
-        tablica[j] = number;
-        printf(" , %d ", tablica[j]);
+
+    printf("Array values: [");
+
+    for (j = 0; j < arraySize; j++) {
+        array[j] = rand() % 50;
+
+        printf("%d", array[j]);
+
+        if (j > 0) {
+            printf(", ");
+        }
     }
-    printf("\n");
-    for (i = 1; i < rozmiarTablicy; i++) {
-        sredniaArytmetyczna = sredniaArytmetyczna + tablica[i];
 
+    printf("]\n");
+
+    for (i = 1; i < arraySize; i++) {
+        arraySum = arraySum + array[i];
     }
 
-    printf("srednia arytmetyczna wynosi %f \n", sredniaArytmetyczna / rozmiarTablicy);
-}
\ No newline at end of file
+    arithmeticAvg = arraySum / arraySize;
```

Comment:
Wcześniej używałeś zmiennej `sredniaArytmetyczna ` która tak na prawdę nie miała wartości średniej tylko sumy elementów z tablicy - błąd merytoryczny, tutaj dodałem zmienną żeby to popraiwć